test/TestUtils/project.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "compile": {
+      "include": [
+        "../../build_projects/shared-build-targets-utils/Utils/DotNetCli.cs",
+        "../../build_projects/shared-build-targets-utils/Utils/EnvVars.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/AnsiConsole.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/AnsiColorExtensions.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/ArgumentEscaper.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/BuildFailureException.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/BuildReporter.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/Command.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/CommandResult.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/Constants.cs",
+        "../../build_projects/Microsoft.DotNet.Cli.Build.Framework/Reporter.cs"
+      ]
+    }
+  },
+
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc3-004338"
```

Comment:
Because:

1. Who is going to update this version?
2. Why not? That is what we are trying to test, so why not use it in as many places as we can.  This is the way CLI tests worked before we split the repo.